### ISSUE TITLE
Traceback if KeyboardInterrupt is emmited in inner process 

### ISSUE THREAD

**Issue by uranusjr** [uranusjr/pyem#5]
```
> pyem py
Python 3.8.0 (tags/v3.8.0:fa919fd, Oct 14 2019, 19:37:50) [MSC v.1916 64 bit (AMD64)] on win32
Type "help", "copyright", "credits" or "license" for more information.
>>>
KeyboardInterrupt
>>> ^Z

Traceback (most recent call last):
  File "C:\Users\uranusjr\AppData\Local\Programs\Python\Python38\lib\runpy.py", line 192, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "C:\Users\uranusjr\AppData\Local\Programs\Python\Python38\lib\runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "C:\Users\uranusjr\AppData\Local\Programs\pipx\bin\pyem.exe\__main__.py", line 7, in <module>
  File "c:\users\uranusjr\appdata\local\programs\pipx\venvs\pyem\lib\site-packages\pyem\__init__.py", line 10, in main
    return dispatch(None)
  File "c:\users\uranusjr\appdata\local\programs\pipx\venvs\pyem\lib\site-packages\pyem\cmds.py", line 182, in dispatch
    return opts.func(project, opts)
  File "c:\users\uranusjr\appdata\local\programs\pipx\venvs\pyem\lib\site-packages\pyem\procs.py", line 157, in run
    return klass(runtime, options.cmd, options.args).run()
  File "c:\users\uranusjr\appdata\local\programs\pipx\venvs\pyem\lib\site-packages\pyem\procs.py", line 123, in run
    return subprocess.call([proc, *self._args], env=self._env)
  File "C:\Users\uranusjr\AppData\Local\Programs\Python\Python38\lib\subprocess.py", line 342, in call
    return p.wait(timeout=timeout)
  File "C:\Users\uranusjr\AppData\Local\Programs\Python\Python38\lib\subprocess.py", line 1079, in wait
    return self._wait(timeout=timeout)
  File "C:\Users\uranusjr\AppData\Local\Programs\Python\Python38\lib\subprocess.py", line 1357, in _wait
    result = _winapi.WaitForSingleObject(self._handle,
KeyboardInterrupt
```

Because it is relayed to parent (pyem). I seem to recall a similar problem when implementing shims for `pythonup`. Is this problem specific to Windows? How do I replicate the implementation for Python? Is it enough to simply catch `KeyboardInterrupt` on `proc.wait()`?

**Comment by uranusjr** [comment]
POSIX seems to react even worse. The parent process exits *immediately* on inner KeyboardInterrupt :(

I guess on POSIX we can just use `os.exec*` to run the child process instead. Did Pipenv have a solution for Windows? I can’t recall… (edit: seems not)

**Comment by uranusjr** [comment]
The POSIX call was fixed in ef8735cfb92f1f1082d52730e24828fdac03655b with `os.execlpe()`.
